extensions/spotify-player/src/search.tsx

```diff
@@ -84,6 +84,15 @@ function SearchCommand({ initialSearchText }: { initialSearchText?: string }) {
                           recentSearchRevalidate();
                         }}
                       />
+                      <Action
+                        icon={Icon.Trash}
+                        title="Remove All Searches"
+                        style={Action.Style.Destructive}
+                        onAction={async () => {
+                          await LocalStorage.setItem("recent-searches", JSON.stringify([]));
```

Comment:
Should we consider adding a [confirmAlert](https://developers.raycast.com/api-reference/feedback/alert#confirmalert) with rememberUserChoice 🙂

Reply:
sure, let me know the appropriate message to display in the alert.

Reply:
let me know if it's necessary to assign keyboard shortcuts as well.

Reply:
We do have `Keyboard.Shortcut.Common.RemoveAll` available 🙂

Reply:
@pernielsentikaer 
updated based on your suggestions. let me know if the Alert message is already sufficient.

<img width="356" alt="Screenshot 2024-08-21 at 14 05 32" src="https://github.com/user-attachments/assets/eca1d105-2740-4f07-aff1-600a211c9bb3">
<img width="347" alt="Screenshot 2024-08-21 at 14 08 08" src="https://github.com/user-attachments/assets/27afa227-f230-4a1b-9099-1c8378254b4b">